web/src/screens/releases/ReleaseFilters.tsx

```diff
@@ -69,18 +69,23 @@ export const IndexerSelectColumnFilter = ({
 }: FilterProps<object>) => {
   const { data, isSuccess } = useQuery(ReleasesIndexersQueryOptions());
 
+  const options = isSuccess && data.map((indexer, idx) => (
+    <FilterOption key={idx} label={indexer.name} value={indexer.identifier} />
+  ));
```

Comment:
I'd move this back to render in the component like previously. It's no reused anywhere so it makes more sense to render like before.

Reply:
Done.